src/basketball/worker/core/GameSim.js

```diff
@@ -226,6 +226,9 @@ class GameSim {
 
 		this.t = g.quarterLength; // Game clock, in minutes
 
+		this.foulRate = g.foulRate;
```

Comment:
Maybe rename foulRate to foulFactor, since it's technically not a "rate" it's a factor that you multiply the base rate by.

Reply:
foulFactor already exists within GameSim.
`		let foulFactor;
		let foulFactor =
		if (g.foulRate > 0) {
			0.65 *
			foulFactor =
			(this.team[this.o].player[p].compositeRating.drawingFouls / 0.5) ** 2;
				0.65 *
				(this.team[this.o].player[p].compositeRating.drawingFouls / 0.5) ** 2 *
				g.foulRate;
		} else {
			foulFactor = 0;
		}`

Reply:
That's okay, g.foulFactor and foulFactor can both exist and be different things. Kind of confusing I guess, but better than calling something a rate when it's not I think.

Reply:
Changed to godModeFoulFactor.